content/02.body.md

```diff
@@ -0,0 +1,282 @@
+## Background
+
+### Node degree bias
+
```

Comment:
May be worth noting that multifunctionality can also produce nodes with high degree. It's unclear if this is a "bias" (and perhaps multifunctional entities are really just inspection-bias exemplars), but there is probably some truth to the idea that participants in multiple processes are more likely to be initially uncovered and studied.

There is some prior work in this area. I am aware of https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0017258 from @gillislab and @PavlidisLab who I am tagging here as they may have helpful suggestions.

Reply:
@gillislab notes this article, which from a very quick read while traveling might be related. He may provide more thoughts when time permits:
https://www.pnas.org/content/116/13/6491/

Reply:
Interesting findings regarding "prior probability of differential expression" which could affect network degree of expression-based networks. I agree we should mention the many reasons why nodes may have different degrees.

Along the same lines, in https://github.com/dhimmel/snplentiful / https://doi.org/10.1161/CIRCGENETICS.115.001181 we found that genes containing more SNPs (i.e. large or hypervariable ones) tended to have higher degree in many networks.